Automattic/wp-calypso
# Build: Move o2 blocks specific Babel config to that package

#### Changes proposed in this Pull Request

Move o2 blocks specific Babel config to that package.

The main motivation for this PR is to make `@automattic/calypso-build` more reusable for consumers, without including stuff that's generally not needed by them. (Even Jetpack needs `@wordpress/import-jsx-pragma'` only for its `extensions/` but not for the React dashboard.)

#### Testing instructions

Verify that o2 blocks still build.

```
npm run distclean
npm ci
npx lerna run prepare --scope="@automattic/o2-blocks"
```

Make sure that the new babel config is actually used. (E.g. by adding a syntax error to `packages/calypso-build/babel.config.js` :sweat_smile: )

Also verify that Calypso still starts and runs properly.
